### DISCUSSION TITLE
Log In screen isn't accurate to 2016

### DISCUSSION THREAD

**Discussion by retrostalgia** [anthony1x6000/ROBLOX2016stylus#20] [Ideas]
Recently noticed that the login in screen wasn't accurate to 2016, so I thought maybe that could be changed at some point? (if possible ofc).

2016 (archive.org) 
![image](https://user-images.githubusercontent.com/111193347/188283346-3fca9a1b-348d-495f-bfce-6a8989c478db.png)

Current (with theme installed) 
![image](https://user-images.githubusercontent.com/111193347/188283368-840f35f2-78b7-41d1-b66a-238b1ebe98ed.png)

**Comment by anthony1x6000** [comment]
I'll try working on this later... It seems possible

**Comment by retrostalgia** [comment]
Good to know. There actually is already another theme that does this exact thing but it breaks every time the 2016 theme is reinstalled, so having this on the actual theme would be nice.

**Reply by anthony1x6000** [reply]
Can u send a link to that? I can probably just steal the code and fix it up a bit to save some time

**Reply by retrostalgia** [reply]
Oh alright, here it is. https://uso.kkx.one/style/227651
The author also has a ton of other 2016 things that aren't in this theme that you could add and fix a little if you'd like, like the old catalog, old promocodes page, old robux page, etc.
